files/build_templates/sonic_debian_extension.j2

```diff
@@ -0,0 +1,115 @@
+
+# Apply apt configuration files
+sudo cp $IMAGE_CONFIGS/apt/sources.list $FILESYSTEM_ROOT/etc/apt/
+sudo cp -R $IMAGE_CONFIGS/apt/sources.list.d/ $FILESYSTEM_ROOT/etc/apt/
+cat $IMAGE_CONFIGS/apt/sonic-dev.gpg.key | sudo LANG=C chroot $FILESYSTEM_ROOT apt-key add -
+
+# Apply environtment configuration files
+sudo cp $IMAGE_CONFIGS/environment/environment $FILESYSTEM_ROOT/etc/
+sudo cp $IMAGE_CONFIGS/environment/motd $FILESYSTEM_ROOT/etc/
+
+# Copy default minigraph
+sudo cp $IMAGE_CONFIGS/minigraph/minigraph_{{sonic_hwsku}}.xml $FILESYSTEM_ROOT/etc/sonic/minigraph.xml
+
+# Copy NTP configuration files and templates
+sudo cp $IMAGE_CONFIGS/ntp/ntp-config.service $FILESYSTEM_ROOT/etc/systemd/system/
+sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable ntp-config.service
+sudo cp $IMAGE_CONFIGS/ntp/ntp-config.sh $FILESYSTEM_ROOT/usr/bin/
+sudo cp $IMAGE_CONFIGS/ntp/ntp.conf.j2 $FILESYSTEM_ROOT/etc/sonic/templates/
+sudo cp $IMAGE_CONFIGS/ntp/ntp.yml $FILESYSTEM_ROOT/etc/sonic/
+
+# Copy rsyslog configuration files and templates
+sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog-config.service  $FILESYSTEM_ROOT/etc/systemd/system/
+sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable rsyslog-config.service
+sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog-config.sh $FILESYSTEM_ROOT/usr/bin/
+sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog.conf.j2 $FILESYSTEM_ROOT/etc/sonic/templates/
+sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog.yml $FILESYSTEM_ROOT/etc/sonic/
+sudo cp $IMAGE_CONFIGS/rsyslog/rsyslog.d/* $FILESYSTEM_ROOT/etc/rsyslog.d/
+
+# Copy interfaces configuration files and templates
+sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config.service  $FILESYSTEM_ROOT/etc/systemd/system/
+sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable interfaces-config.service
+sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config.sh $FILESYSTEM_ROOT/usr/bin/
+sudo cp $IMAGE_CONFIGS/interfaces/*.j2 $FILESYSTEM_ROOT/etc/sonic/templates/
+
+# Copy SNMP configuration files
+sudo cp $IMAGE_CONFIGS/snmp/snmp.yml $FILESYSTEM_ROOT/etc/sonic/
+
+# Generate system desription file
+export git_revision=$(git rev-parse --short HEAD)
+export sonic_hwsku={{sonic_hwsku}}
+export debian_version=`cat $FILESYSTEM_ROOT/etc/debian_version`
+export kernel_version={{kversion}}
+j2 $BUILD_TEMPLATES/sysDescription.j2 > sysDescription 
+sudo mv sysDescription $FILESYSTEM_ROOT/etc/ssw/
+
+# Copy sudoers configuration file
+sudo cp $IMAGE_CONFIGS/sudoers/sudoers $FILESYSTEM_ROOT/etc/
+
+# Copy vendor specific configuration files
+sudo cp -R $VENDOR_CONFIGS/{{sonic_hwsku}}/ $FILESYSTEM_ROOT/etc/ssw/
+
+{% for deb in installer_debs.strip().split(' ') -%}
+sudo dpkg --extract {{deb}} $FILESYSTEM_ROOT
+{% endfor %}
+## Run depmod command for target kernel modules
+sudo LANG=C chroot $FILESYSTEM_ROOT depmod -a {{kversion}}
+{% endif %}
+{% if installer_images.strip() -%}
+{% for image in installer_images.strip().split(' ') -%}
+sudo LANG=C chroot $FILESYSTEM_ROOT docker load < {{image}}
```

Comment:
so, we are doing this docker load in the build phase now. I thought you were thinking doing this in the boot process or onie installation process.